src/main/java/com/gluonhq/richtext/RichTextAreaSkin.java

```diff
@@ -127,6 +129,14 @@ protected RichTextAreaSkin(final RichTextArea control) {
 
     /// PROPERTIES ///////////////////////////////////////////////////////////////
 
+    // textLengthProperty
+    public final ReadOnlyIntegerProperty textLengthProperty() {
+        return viewModel.textLengthProperty();
+    }
+    public final int getTextLength() {
+        return viewModel.getTextLength();
+    }
+
```

Comment:
why do we need this public API on a skin?

Reply:
That was a leftover from previous design - will remove